frappe/desk/reportview.py

```diff
@@ -493,7 +494,11 @@ def delete_bulk(doctype, items):
 		except Exception:
```

Comment:
Move commit above this line to `else` block. If commit causes exception it shouldn't be part of this.

Reply:
Done